op-node/rollup/derive/fuzz_parsers_test.go

```diff
@@ -231,6 +231,23 @@ func FuzzUnmarshallLogEvent(f *testing.F) {
 		f.Add(c.to.Bytes(), b(c.mint), b(c.value), []byte(c.data), c.gasLimit, c.isCreation)
 	}
 
+	// Set the EVM state up once to fuzz against
+	state, err := state.New(common.Hash{}, state.NewDatabase(rawdb.NewMemoryDatabase()), nil)
+	require.NoError(f, err)
+	state.SetBalance(from, uint256.MustFromBig(BytesToBigInt([]byte{0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff})))
+	_, addr, _, err := runtime.Create(common.FromHex(bindings.OptimismPortalMetaData.Bin), &runtime.Config{
+		Origin:   from,
+		State:    state,
+		GasLimit: 20_000_000,
+	})
+	require.NoError(f, err)
+
+	_, err = state.Commit(0, false)
+	require.NoError(f, err)
+
+	portalContract, err := bindings.NewOptimismPortal(addr, nil)
+	require.NoError(f, err)
```

Comment:
Note to self, based on answer from @tynes : create with regular bytecode, rather than assuming previous state setCode, to get rid of "_more" bindings dependency. Nice!